### PR TITLE
Fix image streams

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

File: manifests/05-deployment.yaml [hunk 1]
```diff
@@ -28,3 +28,13 @@ spec:
...
           - name: OPERATOR_NAME
             value: "cluster-logging-operator"
+          - name: ELASTICSEARCH_IMAGE
+            value: "docker.io/openshift/origin-logging-elasticsearch5:latest"
+          - name: FLUENTD_IMAGE
+            value: "docker.io/openshift/origin-logging-fluentd:latest"
+          - name: KIBANA_IMAGE
+            value: "docker.io/openshift/origin-logging-kibana5:latest"
+          - name: CURATOR_IMAGE
+            value: "docker.io/openshift/origin-logging-curator:latest"
```
Comment (richm) [inline manifests/05-deployment.yaml]: curator5

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -121,16 +51,21 @@ func AddOwnerRefToObject(object metav1.Object, ownerRef metav1.OwnerReference) {
...
-	return image.String()
+	env_var_name, ok := COMPONENT_IMAGES[component]
+	if !ok {
+		logrus.Errorf("Environment variable name mapping missing for component: %s", component)
+		return ""
+	}
+	imageTag := os.Getenv(env_var_name)
+	if imageTag != "" {
+		logrus.Errorf("No image tag defined for component '%s' vi environment variable '%s'", component, env_var_name)
+		return ""
```
Comment (ewolinetz) [inline pkg/utils/utils.go]: there isn't a reason to have this return since the one at the end of the function would return `""` in this case

Comment (jcantrill) [reply pkg/utils/utils.go]: Updating to remove the return and noted the condition is in correct.  We should log error if the value is empty

File: pkg/utils/utils.go [hunk 3]
```diff
@@ -121,16 +51,21 @@ func AddOwnerRefToObject(object metav1.Object, ownerRef metav1.OwnerReference) {
...
+	env_var_name, ok := COMPONENT_IMAGES[component]
+	if !ok {
+		logrus.Errorf("Environment variable name mapping missing for component: %s", component)
+		return ""
+	}
+	imageTag := os.Getenv(env_var_name)
+	if imageTag == "" {
+		logrus.Errorf("No image tag defined for component '%s' by environment variable '%s'", component, env_var_name)
+	}
+	logrus.Debugf("Setting component image for '%s' to: '%s'", component, imageTag)
```
Comment (ewolinetz) [inline pkg/utils/utils.go]: is this still needed?

**Comment by jcantrill** [comment]
@smarterclayton Is this how we should expect to ensure we are using released versions of component images?  `ci-operator` will modify manifest files to update operator deployments?

File: pkg/utils/utils.go (hunk 3 above)
Comment (jcantrill) [reply pkg/utils/utils.go]: I think maybe not... but i left it for sanity purposes.  Its only visible at debut level so I thought that would be harmless

**Comment by smarterclayton** [comment]
Yes. The release process will perform a `sed` replacement equivalent from the value you add in `spec.tags.from.name` to the correct image pull spec in your manifests.

**Pull request merged**